.codecov.yml

```diff
@@ -18,4 +18,4 @@ coverage:
         # extension:
         # https://github.com/codecov/browser-extension
         target: auto
-        threshold: 50%
+        threshold: 15%
```

Comment:
This means roughly that new PR need to have at least ~80% (auto) - 15%  = 65% code coverage.